iOS/FVM/FVMTests/DamagedParts/Helpers/HightlightHandlerMock.swift

```diff
@@ -13,7 +13,6 @@
 // limitations under the License.
 
 import SceneKit
-@testable import FVM
```

Comment:
Did you notice that Travis build failed?
Also I tried to testing app in simulator and ... also build failed
You have to restore this line :)

Reply:
This is strange, Xcode tells me that we don't need it

Reply:
Had exactly the same issue but didn't dig into it, might be something with project definition, we can check pbxproj in some spare time